src/store/action/auth.ts

```diff
@@ -1,14 +1,17 @@
+import {AvataaarProps} from "components/Avatar";
```

Comment:
I don't like that we are importing stuff from the components section here. Maybe you can move it to the types section?

Reply:
Would it be okay to move it to `components/Avatar/types`? There are some also the other type definitions for the avataaars (e. g. the different enums for the avatar components).
Or would you rather have it under the `types` directory? If so there would still be a reference to the `components/Avatar/types` path because of the values of the prop, which are located there.